CRITICAL - 2022-02-28 04:22:05 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:22:53 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:25:20 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:26:01 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:27:16 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:27:43 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:28:32 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:40:54 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 04:45:31 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 05:04:33 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(55): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 55, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 05:10:33 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 05:11:38 --> Object of class CodeIgniter\Validation\Validation could not be converted to string
#0 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-02-28 05:12:43 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(89): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 89, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-02-28 05:14:56 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(66): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 05:14:56 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(66): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 05:15:49 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 05:15:49 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 05:19:07 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 05:19:07 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 05:20:29 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 05:20:29 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 06:00:07 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:21:14 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:22:12 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:25:36 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:29:02 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:30:27 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 08:30:42 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-02-28 09:06:41 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:06:41 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 09:07:26 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:07:26 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 09:09:47 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:09:47 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(49): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:15:37 --> Undefined variable: data
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-02-28 09:17:57 --> Undefined variable: arr
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(54): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 54, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-02-28 09:33:12 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(50): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:33:12 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(50): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 09:45:15 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:45:15 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 09:45:55 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 09:45:55 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:10:55 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-02-28 10:10:55 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 10:21:36 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-02-28 10:21:36 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save_user()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 10:23:04 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-02-28 10:23:04 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 10:23:10 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-02-28 10:23:10 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 10:23:43 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2022-02-28 10:23:43 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(52): CodeIgniter\Model->insert(Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 10:25:11 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 10:25:11 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(51): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:26:27 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 10:26:27 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:26:56 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 10:26:56 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:27:30 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 10:27:30 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->save()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:30:41 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\Users.php(183): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2022-02-28 10:30:41 --> SQLite3::exec(): database is locked
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('INSERT INTO `us...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('INSERT INTO `us...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\City_Taxi\app\Controllers\Users.php(183): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#12 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#15 {main}
ERROR - 2022-02-28 10:31:20 --> Email: sendWithSmtp throwed fsockopen(): unable to connect to ssl://smtp.gmail.com:587 (A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
)
CRITICAL - 2022-02-28 10:31:20 --> Undefined property: App\Controllers\Users::$email
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\Users.php(258): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 258, Array)
#1 C:\xampp\htdocs\City_Taxi\app\Controllers\Users.php(186): App\Controllers\Users->sendEmail(Array)
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\Users->register()
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#4 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-02-28 14:19:15 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Views\Admin\userList.php(57): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 57, Array)
#1 C:\xampp\htdocs\City_Taxi\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\City_Taxi\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\City_Taxi\system\Common.php(1089): CodeIgniter\View\View->render('Admin/userList....', Array, true)
#4 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(25): view('Admin/userList....', Array)
#5 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->index()
#6 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#7 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-02-28 14:19:28 --> Array to string conversion
#0 C:\xampp\htdocs\City_Taxi\app\Views\Admin\userList.php(57): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 57, Array)
#1 C:\xampp\htdocs\City_Taxi\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\City_Taxi\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\City_Taxi\system\Common.php(1089): CodeIgniter\View\View->render('Admin/userList....', Array, true)
#4 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(25): view('Admin/userList....', Array)
#5 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->index()
#6 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#7 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-02-28 14:26:22 --> Use of undefined constant uid - assumed 'uid' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 32, Array)
#1 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->getUser()
#2 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#3 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-02-28 23:35:28 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('UPDATE `user_de...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('UPDATE `user_de...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `user_de...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1913): CodeIgniter\Database\BaseConnection->query('UPDATE `user_de...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(298): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(651): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\Model->update('2', Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->updateUser()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 23:36:39 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('UPDATE `user_de...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('UPDATE `user_de...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `user_de...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1913): CodeIgniter\Database\BaseConnection->query('UPDATE `user_de...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(298): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(651): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\Model->update('2', Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->updateUser()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
ERROR - 2022-02-28 23:37:09 --> ErrorException: SQLite3::exec(): database is locked in C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php:115
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\xampp\\htdocs...', 115, Array)
#1 C:\xampp\htdocs\City_Taxi\system\Database\SQLite3\Connection.php(115): SQLite3->exec('UPDATE `user_de...')
#2 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(670): CodeIgniter\Database\SQLite3\Connection->execute('UPDATE `user_de...')
#3 C:\xampp\htdocs\City_Taxi\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `user_de...')
#4 C:\xampp\htdocs\City_Taxi\system\Database\BaseBuilder.php(1913): CodeIgniter\Database\BaseConnection->query('UPDATE `user_de...', Array, false)
#5 C:\xampp\htdocs\City_Taxi\system\Model.php(298): CodeIgniter\Database\BaseBuilder->update()
#6 C:\xampp\htdocs\City_Taxi\system\BaseModel.php(865): CodeIgniter\Model->doUpdate(Array, Array)
#7 C:\xampp\htdocs\City_Taxi\system\Model.php(651): CodeIgniter\BaseModel->update(Array, Array)
#8 C:\xampp\htdocs\City_Taxi\app\Controllers\UserControl.php(53): CodeIgniter\Model->update('11', Array)
#9 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(825): App\Controllers\UserControl->updateUser()
#10 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserControl))
#11 C:\xampp\htdocs\City_Taxi\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\City_Taxi\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\City_Taxi\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
